### PR TITLE
Create SITE.yaml

### PR DESCRIPTION
**Pull request by irodero**
Adding VDC test site 1 for Scientific Computing and Imaging Institute at the University of Utah.

### CODE REVIEW

File: topology/University of Utah/Utah-SCI-VDC1/SITE.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+LongName: Scientific Computing and Imaging Institute VDC test site 1.
+Description: University of Utah's Scientific Computing and Imaging Institute 
+  (SCI) located in Salt Lake City. Test site for the Virtual Data Collaboratory
+  (VDC) initiative.
```
Comment (brianhlin) [inline topology/University of Utah/Utah-SCI-VDC1/SITE.yaml:4]:
```suggestion
LongName: Scientific Computing and Imaging Institute Virtual Data Collaboratory
Description: >
    University of Utah's Scientific Computing and Imaging Institute
    (SCI) located in Salt Lake City. Test site for the Virtual Data
    Collaboratory (VDC) initiative.
```

**Review by brianhlin — COMMENTED** [review]
Hi @irodero thanks for the PR! Would you be able to rename the site from `Utah-SCI-VDC1` to `Utah-SCI-VDC`? We generally mark groups of resources under a site for testing instead of the entire site. For example, at the Center for High Throughput Computing (CHTC), we have separate YAML files for our testing resources vs our production ones:

- https://github.com/brianhlin/topology/blob/project-compat-symlink/topology/University of Wisconsin/CHTC/CHTC-ITB.yaml#L3
- https://github.com/brianhlin/topology/blob/project-compat-symlink/topology/University of Wisconsin/CHTC/CHTC.yaml#L4

**Review by brianhlin — APPROVED** [review]
LGTM, thanks for the updates!

File: topology/University of Utah/Utah-SCI-VDC/SITE.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+LongName: Scientific Computing and Imaging Institute Virtual Data Collaboratory
+ Description: >
+     University of Utah's Scientific Computing and Imaging Institute
+     (SCI) located in Salt Lake City. Test site for the Virtual Data
+     Collaboratory (VDC) initiative.
```
Comment (brianhlin) [inline topology/University of Utah/Utah-SCI-VDC/SITE.yaml:5]:
```suggestion
Description: >
  University of Utah's Scientific Computing and Imaging Institute
  (SCI) located in Salt Lake City. Test site for the Virtual Data
  Collaboratory (VDC) initiative.
```

Fix spacing

File: topology/University of Utah/Utah-SCI-VDC1/SITE.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+ID:
```
[the remark this answers is not in the document]
Comment (irodero) [reply topology/University of Utah/Utah-SCI-VDC1/SITE.yaml]: Thank you very much for processing and fixing the new site request.

**Comment by irodero** [comment]
> Hi @irodero thanks for the PR! Would you be able to rename the site from `Utah-SCI-VDC1` to `Utah-SCI-VDC`? We generally mark groups of resources under a site for testing instead of the entire site. For example, at the Center for High Throughput Computing (CHTC), we have separate YAML files for our testing resources vs our production ones:
> 
> * https://github.com/brianhlin/topology/blob/project-compat-symlink/topology/University of Wisconsin/CHTC/CHTC-ITB.yaml#L3
> * https://github.com/brianhlin/topology/blob/project-compat-symlink/topology/University of Wisconsin/CHTC/CHTC.yaml#L4

Thank you, Brian. It does make completely sense. I have updated the directory name and added a resource group for a test origin. Your assistance is greatly appreciated.

**Comment by irodero** [comment]
Dear @brianhlin, it looks like the checks are failing. Please let me know if I need to update any of the two yaml files. I left the groupID empty in the resource group yaml file. Thank you very much for your assistance.

File: topology/University of Utah/Utah-SCI-VDC/Utah-SCI-VDC-ITB.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+GroupDescription: Resource group for the SCI VDC ITB site
+GroupID:
```
Comment (brianhlin) [inline topology/University of Utah/Utah-SCI-VDC/Utah-SCI-VDC-ITB.yaml:2]:
```suggestion
GroupID:  1336
```

File: topology/University of Utah/Utah-SCI-VDC/Utah-SCI-VDC-ITB.yaml [hunk 5]
```diff
@@ -0,0 +1,26 @@
+GroupDescription: Resource group for the SCI VDC ITB site
+GroupID: 
+Production: false
+SupportCenter: Self Supported
+Resources:
+  Utah-SCI-VDC-ITB-ORIGIN:
+    Active: true
+    Description: >-
+        This is a testing OSDF origin server on a VDC resource.
+    ID:
```
Comment (brianhlin) [inline topology/University of Utah/Utah-SCI-VDC/Utah-SCI-VDC-ITB.yaml:10]:
```suggestion
    ID: 1401
```

**Review by brianhlin — COMMENTED** [review]
Adding IDs

File: topology/University of Utah/Utah-SCI-VDC/SITE.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+  (SCI) located in Salt Lake City. Test site for the Virtual Data
+  Collaboratory (VDC) initiative.
+ ID: 10348
+ AddressLine1: 72 S Central Campus Drive, Room 3750
+ City: Salt Lake City
+ Country: United States
+ State: UT
+ Zipcode: '84112'
+ Latitude: 40.7608
+ Longitude: -111.891
```
Comment (brianhlin) [inline topology/University of Utah/Utah-SCI-VDC/SITE.yaml:13]:
```suggestion
ID: 10348
AddressLine1: 72 S Central Campus Drive, Room 3750
City: Salt Lake City
Country: United States
State: UT
Zipcode: '84112'
Latitude: 40.7608
Longitude: -111.891
```

**Comment by irodero** [comment]
Hi @brianhlin, I believe the checks have passed. Is this ready to be merged? I accidentally closed the PR and re-opened it. Thank you!

**Comment by brianhlin** [comment]
Yup, we're all good. Thanks again for the PR!

**Pull request merged**